packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  hiddenContentStyle,
+  HighContrastSelector,
+} from '../../Styling';
+import { RatingSize, IRatingStyleProps, IRatingStyles } from './Rating.types';
+
+export function getStyles(props: IRatingStyleProps): IRatingStyles {
+  const {
+    disabled,
+    theme
+  } = props;
+
+  const {
+    semanticColors,
+    palette
```

Comment:
are you actually using "palette" here? it should be giving you an unused lint error